src/dmd/backend/gsroa.d

```diff
@@ -5,27 +5,39 @@
  * Copyright:   Copyright (C) 2016-2018 by The D Language Foundation, All Rights Reserved
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/gslice.c, backend/gslice.c)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/gsroa.c, backend/gsroa.d)
  */
 
+module backend.gsroa;
```

Comment:
This should reside in the `dmd` package like all the other D files in the `backend` package.

Reply:
ok